extensions/panache/hibernate-orm-rest-data-panache/deployment/src/test/java/io/quarkus/hibernate/orm/rest/data/panache/deployment/AbstractDevModeTest.java

```diff
@@ -4,10 +4,12 @@
 import static io.restassured.RestAssured.when;
 import static org.assertj.core.api.Assertions.assertThat;
 
+import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 
 import io.restassured.response.Response;
 
+@Tag(TestTags.DEVMODE)
 public abstract class AbstractDevModeTest {
```

Comment:
The `@Tag` annotation is `@Inherited`, so I expect it'll apply to all test subclasses.